addition.c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+int main()
+{
+    int a, b, sum;
```

Comment:
Never declare variables like a & b.  The variable should be declared with some meaningful names.

Reply:
Variable name has been changed to firstnum & secondnum